lib/ansible/module_utils/facts/network/linux.py

```diff
@@ -173,7 +173,8 @@ def parse_ip_output(output, secondary=False):
                         if '/' in words[1]:
                             address, netmask_length = words[1].split('/')
                             if len(words) > 3:
-                                broadcast = words[3]
+                                if words[2] == 'brd':
```

Comment:
I'm going back and forth on this: Does defaulting `broadcast` to `None` make more sense instead of `''` since it won't always exist? Then it would serialize to `null`, which is probably more semantically accurate.

Reply:
Semantically it's probably more correct, but in practice, dealing with different types in playbooks is a pain. Keeping a consistent type is easier on playbook authors.